R/read_idats.R

```diff
@@ -49,6 +49,12 @@ read_idats <- function(idat_files,quiet=FALSE){
         manifest = data.table::copy(ewastools:::manifest_epic)
         controls = data.table::copy(ewastools:::controls_epic)
     }
+    else if(P==1105209)
+	{
+		platform="EPIC_v2"
+        manifest = data.table::copy(ewastools:::manifest_epic_v2)
+        controls = data.table::copy(ewastools:::controls_epic_v2)
+	}
```

Comment:
```suggestion
    {
        platform="EPIC_v2"
        manifest = data.table::copy(ewastools:::manifest_epic_v2)
        controls = data.table::copy(ewastools:::controls_epic_v2)
     }
```
Indentation
Tip: better use spaces instead of tabs so that the alignment is the same across editors. Most text editors enable automatic conversion of tabs to spaces.